src/material-experimental/mdc-button/_button-theme.scss

```diff
@@ -316,13 +316,6 @@ $mat-fab-state-target: '.mdc-fab__ripple';
       @include mdc-ripple-theme.states(
           $query: mdc-helpers.$mat-theme-styles-query, $ripple-target: $mat-button-state-target);
 
-      &.mat-unthemed {
```

Comment:
Seems like this was added by you in https://github.com/angular/components/pull/16986. I don't remember the context, but maybe there was a specific reason for it? If that's not the case, should we remove the class binding as well?

Reply:
All the buttons apply MDC's purple color by default, except for the icon button which has `color: inherit`. In the other cases, they do need this "unthemed" style to make sure they override the purple. I think this is to just make sure the spec is matched since icon buttons show up on the toolbar, but other buttons don't